pages.it/common/7z.md

```diff
@@ -10,7 +10,7 @@
 
 `7z a {{archivio_criptato.7z}} -p {{password}} -mhe {{archivio.7z}}`
 
-- Estrai un archivio mantenendo la gerarchia delle cartelle:
+- Estrai un archivio mantenendo la gerarchia delle directory:
```

Comment:
Is "directory" the same word in Italian?

Reply:
Not an italian word, but an acquired english word. Much like "skateboard" or "file". While "cartella" just means "folder".